target_statistics: [2, 1, 1, 1, 3]